tests/unit/LicenseDropdown.spec.js

```diff
@@ -0,0 +1,68 @@
+import { mount, createLocalVue, config } from "@vue/test-utils";
+import LicenseDropdown from "@/Components/LicenseDropdown.vue";
+import VueI18n from "vue-i18n";
+import Vuex from "vuex";
+import Buefy from "buefy";
+
+describe("LicenseDropdown.vue", () => {
+    let wrapper;
+    let getters;
+    let store;
+    let state;
+
+    beforeEach(() => {
+        const localVue = createLocalVue();
+        localVue.use(VueI18n);
+        localVue.use(Vuex);
+        localVue.use(Buefy);
+
+        getters = {
+            shortName: state => {
+                return "name";
+            },
+
+            fullName: state => {
+                return "fullName";
+            }
+        };
+        store = new Vuex.Store({
+            store,
+            getters
+        });
+        const messages = require("@/locales/en.json");
+        const i18n = new VueI18n({
+            locale: "en",
+            fallbackLocale: "en",
+            messages: messages
+        });
+        config.mocks.i18n = i18n;
+        config.mocks.$t = key => {
+            return i18n.messages[key];
+        };
+        wrapper = mount(LicenseDropdown, {
+            localVue,
+            store
+        });
+    });
+
+    it("Has the field tag", () => {
```

Comment:
When testing multiple markup features (field tag, option tag, etc) it is better to use snapshot testing, because it checks for multiple features in one test.